compose/project.py

```diff
@@ -205,6 +215,27 @@ def kill(self, service_names=None, **options):
     def remove_stopped(self, service_names=None, **options):
         parallel.parallel_remove(self.containers(service_names, stopped=True), options)
 
+    def initialize_volumes(self):
+        try:
+            for volume in self.volumes:
+                volume.create()
+        except NotFound:
+            raise ConfigurationError(
+                'Volume %s specifies nonexistent driver %s' % (volume.name, volume.driver)
+            )
+        except APIError as e:
+            if 'Choose a different volume name' in str(e):
```

Comment:
@dnephin @shin- have you opened an issue for that in docker/docker? Sounds like a good improvement (I think I've seen a discussion around this, so there *may* already be an issue for it)